src/handler.js

```diff
@@ -11,6 +11,10 @@ const BUCKET = process.env.DESTINATION_BUCKET;
 const SOURCE_BUCKET = process.env.SOURCE_BUCKET;
 const LOCAL = process.env.AWS_SAM_LOCAL;
 const UPLOAD_KEY = 'videos';
+const HEADERS = {
+  'Content-Type': 'application/json',
+  'Access-Control-Allow-Origin': '*'
```

Comment:
I think we'll also want `'Access-Control-Allow-Headers': 'Content-Type'`, and possibly also an `'Access-Control-Allow-Methods'`

Reply:
I've already tested/verified that adding `'Access-Control-Allow-Origin': '*'` here alone fixed the specific CORS issue reproduced in the earlier test, so the extra headers might need additional testing that i'll leave to you.